draft-ietf-core-dns-over-coap.md

```diff
@@ -267,8 +267,11 @@ The construction algorithm for DoC requests is as follows, going through the pro
   As a fallback, an address MAY be queried for the target name of the SVCB record.
 - The destination port for the request MUST be taken from the "port" SvcParam value, if present.
   Otherwise, take the default port of the CoAP transport.
-- The target name of the SVCB record MUST be set in the Uri-Host option if the resolved address for the target name differs from the destination address.
-  Otherwise, the Uri-Host option MAY be set from the target name.
+- The request URI's hostname component MUST be the Authentication Domain Name (ADN) when obtained through DNR
+  and MUST be the target name of the SVCB record when obtained through a `_dns` query
+  (if AliasMode is used, to the target name of the AliasMode record).
+  This is usually achieved by setting that name in TLS Server Name Indication (SNI) {{?RFC8446}},
```

Comment:
"usually" sounds a bit vague to me.

Reply:
Changed to:

>   This can be achieved efficiently by setting that name in TLS Server Name Indication (SNI) {{?RFC8446}},
  or by setting the Uri-Host option on each request.

which does not take any position on how precisely this interacts with DTLS.